contract/tests/test_contract.py

```diff
@@ -1,83 +1,129 @@
-# Copyright 2016 Tecnativa - Carlos Dauden
-# Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2018 Tecnativa - Carlos Dauden
+# Copyright 2018 Tecnativa - Pedro M. Baeza
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from datetime import timedelta
+from dateutil.relativedelta import relativedelta
 from odoo import fields
 from odoo.exceptions import ValidationError
 from odoo.tests import common
 
 
+def to_date(date):
+    return fields.Date.to_date(date)
```

Comment:
For not needing to define this method:
```python
from fields.Date import to_date
```

Reply:
I'm afraid we can't to that.

`ModuleNotFoundError: No module named 'odoo.fields.Date'; 'odoo.fields' is not a package`